Update PR#4 (#8)
elong0527/r4csr: tlf-disposition.Rmd

# Disposition {#disposition}

Following [ICH E3 guidance](https://database.ich.org/sites/default/files/E3_Guideline.pdf), 

> we need to summarize the accounting of all participants who entered the study in Section 10.1, Disposition of Participants.

Reviewer: a summary needs to be provided to account for

Author: ```suggestion
A summary needs to be provided to account for of all participants who entered the study in Section 10.1, Disposition of Participants.
```

In the code below, we calculate the number and percentage of participants who complete the study.
We split `n_reason` because we need to customize the row order of the table.

```{r}
n_complete <- n_reason %>% 
  filter(row == "Completed")

n_complete
```

In the code below, we calculate the number and percentage of participants who discontinued the study for different reasons.

> For display purposes, we use `paste0("    ", row)` to add leading spaces to mimick identation in the final report.

Reviewer: For display purposes,  `paste0(" ", row)` is used to add leading spaces to mimick identation in the final report

Author: ```suggestion
For display purposes, `paste0("    ", row)` is used to add leading spaces to mimick identation in the final report.
```

tbl_disp
```

In the below code, formatting of the final table is defined. 

> We highlighted items that is not discussed in the previous discussion.

Reviewer: Items that were not discussed in the previous sections, are highlighted below.

Author: ```suggestion
Items that were not discussed in the previous sections, are highlighted below.
```
